From 093b7ed8224f80a1f84a64b35bef36f7c3567054 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 10 Jan 2020 17:24:52 -0800
Subject: [PATCH 0799/1851] dt-bindings: dma: xilinx_frmbuf: Add support for 8
 ppc

commit 588696ad4a2d88c72117d9beb900e1082550bd28 from
https://github.com/Xilinx/linux-xlnx.git

Add 8 as a valid option for pixels per clock as the IP now supports it.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
index aa068f68a8c0..484389930dca 100644
--- a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
+++ b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
@@ -33,7 +33,7 @@ Required Properties Common to both FB Read and FB Write:
 			  See VIDEO FORMATS table below and examples.
 
 Required Properties Common to both FB Read and FB Write for v2.1:
-- xlnx,pixels-per-clock	: Pixels per clock set in IP (1, 2 or 4)
+- xlnx,pixels-per-clock	: Pixels per clock set in IP (1, 2, 4 or 8)
 - clocks: Reference to the AXI Streaming clock feeding the AP_CLK
 - clock-names: Must have "ap_clk"
 - xlnx,max-height	: Maximum number of lines.
-- 
2.31.1

